src/services/userAuthentication.js

```diff
@@ -28,22 +24,67 @@ class UserAuthentication {
         throw err
       } else {
         that.username=res['user'];
-        return that.findOrCreateUser(that.username);
+        return that.userInOurDb = that.findOrCreateUser(that.username);
       }
     });
   }
+  followSteemUser(steemUserNameToFollow) {
+    let that=this;
```

Comment:
`let self = this;` tends to be js convention

Reply:
another thing, if a variable isn't re-assigned, use `const`. If it is re-assigned, use `let`

Reply:
cool